modules/admin_manual/pages/release_notes.adoc

```diff
@@ -19,6 +20,160 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.2.0
+
+Dear ownCloud administrator, please find, below, the changes and known issues in ownCloud Server 10.2 that need your attention. 
+You can also read https://owncloud.org/changelog/server/[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Migrations
+
+Please note that this minor release contains database migrations which impact the upgrade duration.
+Specifically:
+
+* The `oc_share` table has a new column. The impact of this change depends on the number of shares in your ownCloud installation.
+* The `oc_authtoken` table's login name column size has been increased. The impact depends on the number of recently logged in users plus several app passwords.
+
+=== PHP 5.6/7.0 Deprecation
+
+Following up the https://doc.owncloud.com/server/admin_manual/release_notes.html#php-5-6-deprecation[PHP 5.6/7.0 deprecation notice in the ownCloud Server 10.0.8 releases] ownCloud Server 10.2 *does not support PHP 5.6* and some apps no longer support older PHP versions. 
+Additionally, the next, minor, version of ownCloud Server will support PHP 7.3.
```

Comment:
The next minor version? Hmm, I actually would like to avoid such promises.

Reply:
Yes, guessing whether a version will happen to be patch, minor or major results in confusion for people when the guess turns out wrong.
Maybe:
"PHP 7.3 support will be in a soon-to-be-released version"
(we cannot say "next" because there might be a 10.2.1 patch to fix some issue/CVE or... and we do not know about that sort of thing)
or remove this statement?

Reply:
How about being just a bit more non-specific and say "_PHP 7.3 support will be available in an upcoming version_"?